src/plugins/navigation/public/top_nav_menu/_index.scss

```diff
@@ -1,7 +1,7 @@
 .kbnTopNavMenu__wrapper {
   z-index: 5;
 
-  .kbnTopNavMenu {      
-    padding: $euiSizeS 0px $euiSizeXS;
+  .kbnTopNavMenu {
+    padding: $euiSizeS 0px $euiSizeXS $euiSizeS;
```

Comment:
This component is reused across Kibana. Introducing this padding will mess up things in other apps - Discover / Visualize.